docs/ruleset_functions/get_building_lab_zones.md

```diff
@@ -0,0 +1,34 @@
+# get_building_lab_zones  
+
+**Schema Version:** 0.0.28
+
+**Description:** returns a list of all of the zones in the building that include a laboratory space
+
+**Inputs:** 
+- **P_RMI**
+
+**Returns:**  
+- **laboratory_zones_list**: a list of zone.ids for all zones that have a laboratory space in the building
+ 
+**Function Call:**
+
+design_airflow
+## Logic:
+- set the building_total_lab_exhaust variable to 0: `building_total_lab_exhaust = 0`
```

Comment:
The function just looks at the lighting space type, so seems like line 17 should be deleted?